tests/functional/object/mcg/test_bucket_replication_and_its_metrics.py

```diff
@@ -0,0 +1,208 @@
+import logging
+import time
+import json
+import pytest
+
+
+from ocs_ci.framework.pytest_customization.marks import (
+    mcg,
+    polarion_id,
+    red_squad,
+    runs_on_provider,
+    tier2,
+)
+from ocs_ci.framework.testlib import MCGTest
+from ocs_ci.framework import config
+from ocs_ci.ocs.bucket_utils import (
+    update_replication_policy,
+    wait_for_object_versions_match,
+    copy_random_individual_objects,
+)
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.mcg_replication_policy import McgReplicationPolicy
+from ocs_ci.utility.prometheus import PrometheusAPI
+from ocs_ci.ocs import constants
+
+logger = logging.getLogger(__name__)
+
+
+TIMEOUT = 300
```

Comment:
I dont see script is using this value anywhere. Please remove it if not needed

Reply:
yep, I forgot to remove that, will update it